SimG4CMS/HcalTestBeam/interface/HcalTB06Analysis2015.h

```diff
@@ -0,0 +1,58 @@
+#ifndef SimG4CMS_HcalTestBeam_HcalTB06Analysis2015_H
```

Comment:
@davidlange6 , the code was changed completely due to several reasons and we did not want disturb initial testbeam code - we are doing a new analysis. I would agree with your proposal to substitute completely old code if we fully proof that we do things correctly. It is not yet the case. So, may be better to release an alternative class now and in the following PR replace the old one, however, I can update this PR also.